CMakeLists.txt

```diff
@@ -1,75 +1,85 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.10.2)
 project(moveit2_tutorials)
 
-set(CMAKE_CXX_STANDARD 14)
+if(NOT "${CMAKE_CXX_STANDARD}")
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-find_package(catkin REQUIRED
-  COMPONENTS
-    interactive_markers
-    moveit_core
-    moveit_ros_planning
-    moveit_ros_planning_interface
-    moveit_ros_perception
-    rviz_visual_tools
-    moveit_visual_tools
-    pluginlib
-    geometric_shapes
-    pcl_ros
-    pcl_conversions
-    rosbag
-    tf2_ros
-    tf2_eigen
-    tf2_geometry_msgs
-)
-
 find_package(Eigen3 REQUIRED)
 find_package(Boost REQUIRED system filesystem date_time thread)
+find_package(ament_cmake REQUIRED)
+find_package(moveit_core REQUIRED)
+find_package(moveit_ros_planning REQUIRED)
+find_package(moveit_ros_planning_interface REQUIRED)
+find_package(moveit_ros_perception REQUIRED)
+find_package(interactive_markers REQUIRED)
+#find_package(rviz_visual_tools REQUIRED)
+#find_package(moveit_visual_tools REQUIRED)
+find_package(geometric_shapes REQUIRED)
+#find_package(pcl_ros REQUIRED)
+#find_package(pcl_conversions REQUIRED)
+#find_package(rosbag REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_action REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(tf2_ros REQUIRED)
+find_package(tf2_eigen REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
 
 set(THIS_PACKAGE_INCLUDE_DIRS
   doc/interactivity/include
 )
 
-catkin_package(
-  LIBRARIES
-  INCLUDE_DIRS
-  CATKIN_DEPENDS
-    moveit_core
-    moveit_visual_tools
-    moveit_ros_planning_interface
-    interactive_markers
-    tf2_geometry_msgs
-  DEPENDS
-    EIGEN3
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+  ament_cmake
+  rclcpp
+  rclcpp_action
+  tf2_geometry_msgs
+  tf2_ros
+  moveit_core
+  # rviz_visual_tools
+  # moveit_visual_tools
+  moveit_ros_planning_interface
+  interactive_markers
+  tf2_geometry_msgs
+  moveit_ros_planning
+  pluginlib
+  Eigen3
+  Boost
 )
 
-###########
-## Build ##
-###########
-
 include_directories(${THIS_PACKAGE_INCLUDE_DIRS})
-include_directories(SYSTEM ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIR} ${EIGEN3_INCLUDE_DIRS})
 
-add_subdirectory(doc/controller_configuration)
+# add_subdirectory(doc/controller_configuration)
 add_subdirectory(doc/hand_eye_calibration)
```

Comment:
Should we remove the empty CMakeLists.txt files in this package? The only reason this one is uncommitted is because it has an cmake file.
[hand_eye_calibration](https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/hand_eye_calibration/CMakeLists.txt)
[planning](https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/planning/CMakeLists.txt)

Reply:
I guess we should remove all empty cmake files, don't know why we would have them in the first place